.travis.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     - env:
         - JDK="adopt@~1.11-0"
         - _JAVA_OPTIONS="-XX:+UnlockExperimentalVMOptions -XX:+UseJVMCICompiler"
-        - CMD="examples/it:test"
+        - CMD="examples/test; examples/it:test"
```

Comment:
`examples/test` is already included in the general test stage, no?

Reply:
Correct. reverting.